test/jobs/analytics_job_test.rb

```diff
@@ -128,9 +128,9 @@ def teardown
     snapshots = CacheAnalyticsJob.all_monitoree_snapshots(1, @@monitorees, 1)
     verify_snapshot(snapshots, 0, 'Last 24 Hours', 3, 0, 2, 0)
     verify_snapshot(snapshots, 1, 'Last 24 Hours', 2, 0, 0, 0)
-    verify_snapshot(snapshots, 2, 'Last 7 Days', 14, 0, 1, 0)
+    verify_snapshot(snapshots, 2, 'Last 7 Days', 13, 0, 1, 0)
```

Comment:
I saw this test what seemed to be intermittently failing. Is this actually the necessary change and are we sure there's not something with the analytics updates that recently went into `1.18.0`?

Reply:
yeah I can change this back, I changed it before realizing it was an intermittent failure

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/1f13143f8968feff3c7e9ada6139b47e5cd4ecff#diff-b5c61075f4a26ce671654d53d4badeb73e82cd4c1144e122ddfcf988ed198ab6R131